CONTRIBUTING-students.md

```diff
@@ -6,44 +31,37 @@ The 2016 student application window is March 14th to 25th.
 [The Python Software Foundation Wiki](https://wiki.python.org/moin/SummerOfCode/2016#Students)
 lists a few things you can do to get started in free and open source software.
 
-## Getting Started
-
-**Please use the instructions of the project/organization
-you are interested in work with.**
-
-## biocore
-
-Read [default instructions](#default-instructions).
-
-## DyND
-
-Read [default instructions](#default-instructions).
-
-## Gensim
-
-Read [default instructions](#default-instructions).
-
-## JuliaOpt
+## Am I experienced enough?
 
-Read [default instructions](#default-instructions).
+The answer is generally: **Yes**. 
+We value creativity, intelligence and enthusiasm above specific knowledge of the libraries or algorithms we use. 
+We think that an interested and motivated student who is willing to learn is more valuable than anything else.
+The range of available projects should suit people with different backgrounds, both with and without prior experience of computational chemistry.
+At the same time if you have experience using your project of choice or one of it's dependencies (e.g. language) make sure to let us know about that as well.
 
-## matplotlib
+## Our Expectations From Students
 
-Read [default instructions](#default-instructions).
+**Communication**
 
-## Pandas
+- Write a short report for us every second week, this can be a blog post
+- Commit early and commit often! Push to a public repository (e.g. github) so that we can see and review your work.
+- Actively work on our project timeline and communicate with us during the community bonding period
+- Communicate every working day with your mentor. Just say "Hello" if you like. It can be via email, skype, github comments, etc
+- If there is a reason why you can't work or can't contact us on a regular basis please make us aware of this.
+- If you don't communicate with us regularly we will fail you.
 
-Read [default instructions](#default-instructions).
+**Midterm and Final evaluations**
 
-## Software Carpentry
+- Set a realistic goal for mid-term. If you fail to meet your own goal we are more likely to fail you in the evaluations
+- Have some code merged into the current development branch of your project at the end of the summer to pass the final evaluation
```

Comment:
I changed it to `Have at least one commit in the development branch`. We created this in other projects I worked with because we feel it is important for a student to see that is work is valued and being able to show of your name in a contributer list after the summer belongs to that. It should also enforce that the chosen projects are divided into sub projects and that the whole projects can be done in the summer. If this isn't possible for a project we can make an exception.

I also normally interpret this very freely and most students I accepted fulfilled this requirement during the application phase by fixing a bug. This is also why I explicitly don't mention their own project in this sentence.